tests/publiccloud/run_ltp.pm

```diff
@@ -86,6 +86,7 @@ sub run {
     my $ltp_repo = get_var('LTP_REPO', 'https://download.opensuse.org/repositories/benchmark:/ltp:/stable/' . generate_version("_") . '/');
     my $provider;
     my $instance;
+    my $ltp_package = 'ltp-stable';
```

Comment:
We use `LTP_PKG` variable  https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/1a35043d7dac67f2f376d78ac4da1c78173196f3/tests/kernel/install_ltp.pm#L276  or default.

Reply:
this tests are running for 2-3 years and during this period there was never the case that I need to use different package here. On other side we have sooo many variables so each another variable especially when there is 0 chances  to actually use this variable